experimenter/experimenter/nimbus-ui/src/components/PageEditMetrics/FormMetrics/index.tsx

```diff
@@ -128,6 +128,59 @@ const FormMetrics = ({
     [isLoading, onSave, handleSubmit, primaryOutcomes, secondaryOutcomes],
   );
 
+  // Two new variables to handle the primary and secondary states.
+  const [valid, setValid] = useState({ primary: false, secondary: false });
+
+  const primaryContainerDivRef = useRef<HTMLDivElement>(null);
+  const secondaryContainerDivRef = useRef<HTMLDivElement>(null);
+
+  useEffect(() => {
+    const primaryContainerDiv = primaryContainerDivRef.current;
+    const secondaryContainerDiv = secondaryContainerDivRef.current;
+
+    if (primaryContainerDiv) {
+      const valueContainer = primaryContainerDiv.getElementsByClassName(
+        "css-g1d714-ValueContainer",
+      )[0];
+      const valueContainerDiv =
+        primaryContainerDiv.getElementsByClassName("css-yk16xz-control")[0];
+      if (valueContainer) {
+        valueContainer.classList.toggle("form-control", valid.primary);
+        valueContainer.classList.toggle("is-valid", valid.primary);
+        valueContainer.classList.toggle("border-0", valid.primary);
+        valueContainerDiv?.classList.toggle("border-success", valid.primary);
+      }
+    }
+
+    if (secondaryContainerDiv) {
+      const valueContainer = secondaryContainerDiv.getElementsByClassName(
+        "css-g1d714-ValueContainer",
+      )[0];
```

Comment:
here you are using container class, which can change, I would suggest using the ids concept, like try to use getElementById() instead of containers id

Reply:
Sure. I will work on that asap. Thanks

Reply:
The HTML elements for the `Select` component are a couple of `divs`. Below is a screenshot of the `divs`. It only has classes. I am not sure `getElementById` is possible.

<img width="416" alt="Screen Shot 2023-03-29 at 3 52 43 PM" src="https://user-images.githubusercontent.com/105272189/228603430-3d92bb95-cd53-4d87-9201-f4e82a66be09.png">